src/components/Footer.ts

```diff
@@ -22,19 +25,55 @@ export class Footer extends LitElement {
         color: var(--bkd-func-fg-black);
       }
 
+      footer {
+        display: flex;
+        justify-content: space-between;
+      }
+
       .copyright {
         font-size: 0.8125rem;
         font-weight: 300;
         letter-spacing: 0.02rem;
         word-spacing: 0.05rem;
       }
 
+      .footer-nav a {
+        font-size: 14px;
+        font-weight: 400;
+        letter-spacing: 0.01rem;
+        word-spacing: 0.025rem;
+        line-height: 1.5;
+        color: var(--bkd-func-fg-black);
+        text-decoration: none !important;
+        display: inline-block;
+        margin: 0 calc(0.5vw + 30px) 0 0;
+      }
+
+      a:after {
+        display: block;
+        content: "";
+        border-bottom: 2px solid var(--bkd-func-fg-black);
+        transform: scaleX(0);
+        transition: transform 100ms ease-in-out;
+      }
+
+      a:hover::after,
+      a:focus::after,
+      a:active::after {
+        transform: scaleX(1);
+      }
+
       /* Medium screen */
 
       @media screen and (max-width: 1200px) {
         :host {
           --bkd-footer-padding-horizontal: var(--bkd-margin-horizontal-medium);
         }
+
+        footer {
+          flex-direction: column-reverse;
+          gap: 20px;
```

Comment:
Würde ich auch in `rem` angeben, damit bei veränderter Basisschriftgrösse der Abstand passen würde.